### PR TITLE
Issue #809/#799 fix content in section 6.4 and 6.2

### PR DESCRIPTION
**Pull request by 82817**
In section 6.4 Definitions of Data Deception and Graphic Integrity, I've - 

1. Taken out the first two lines of the first paragraph in 6.4.
2. Starting from "More people than ever..." to the end of the bullet points (right before the table), is kept in this section. Also renamed as "Data Deception and Data Integrity"
3. The table on the bottom of 6.4 describes ways an improper graph might be created. I've put this in section 6.3.3 Design
4. The whole section 6.2 is out of context for ethics. I've put it after 3.6.1 Data Mining and Data Visualization and named it Implications of Good Data Visualization

### CODE REVIEW

**Comment by mayjai** [comment]
Great! I think the content makes more sense after the reordering.

**Comment by syragrg** [comment]
The restructuring is more logical now, thanks for the changes!

File: 02-fundamentals.Rmd [hunk 1]
```diff
@@ -436,6 +436,20 @@ There are other examples and use cases that may be downloaded, and the scripts a
...
+
+We should understand how to best derive meaning from data, but first we should understand why its presentation in graphical format is so powerful. Furthermore, while the ideal purpose of data visualization is to facilitate understanding of data, visualization can also be used to mislead. Some of the main methods of doing so are omitting baselines, axis manipulation, omitting data, and ignoring graphing convention. Examples of these methods will be explored later in this chapter.
+
+
+**SNo.** | **Principle** | **Description**
+---------|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------
+**1.** | **Easy Recall** | People can process images quicker than words. When data is transformed into images, the readability and cognition of the content greatly improves. While people can only remember just 10% of what they hear and 20% of what they read, retention jumps up to 80% for visual information with interaction.
```
Comment (syragrg) [inline 02-fundamentals.Rmd]: I deleted the line starting from 'while people...' previously. Not sure if im understanding this right but did you pull an old branch?

Comment (82817) [reply 02-fundamentals.Rmd]: You are correct. I forgot to create a new branch. I'm going to close this pull request and duplicate the information on the new one

**Pull request closed without merging**